README.md

```diff
@@ -2,7 +2,7 @@
 
 This is the Kubernetes Ingress Controller for Caddy. It includes functionality
 for monitoring `Ingress` resources on a Kubernetes cluster and includes support
-for providing automatic HTTPS certificates for all hostnames defined in ingress
+for providing automatic HTTPS certificates for all hostnames defined in the ingress
```

Comment:
Another small changes here, we go over 80 chars in this line, can you move the last word to the next line 😅

Reply:
Done 😅